src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs

```diff
@@ -263,6 +263,49 @@ public string CreateOffer(Bundle bundle)
             return offer;
         }
 
+        /// <summary>
+        /// Creates SDP offer asynchronously to start a new WebRTC connection to a remote peer.
+        /// </summary>
+        /// <remarks>The WebRTC must be in the <see cref="WebRTCState.Negotiating"/></remarks>
+        /// <returns>The SDP offer.</returns>
+        /// <exception cref="InvalidOperationException">The WebRTC is not in the valid state.</exception>
+        /// <exception cref="ObjectDisposedException">The WebRTC has already been disposed.</exception>
+        /// <seealso cref="CreateOffer(Bundle)"/>
+        /// <since_tizen> 9 </since_tizen>
+        public async Task<string> CreateOfferAsync() => await CreateOfferAsync(null);
+
+        /// <summary>
+        /// Creates SDP offer with option asynchronously to start a new WebRTC connection to a remote peer.
+        /// </summary>
+        /// <remarks>The WebRTC must be in the <see cref="WebRTCState.Negotiating"/></remarks>
+        /// <param name="bundle">Configuration options for the offer.</param>
```

Comment:
In native API, we mention the bundle does not affect like below. We better mention it similarly. 
=> "The @a options currently has no effect."

Or, we don't need to add the function with the bundle parameter for now.

Reply:
I missed it. I'll remove the API with `Bundle`.